src/Features/Core/Portable/CodeRefactorings/CodeRefactoringContextExtensions.cs

```diff
@@ -20,12 +21,32 @@ internal static class CodeRefactoringContextExtensions
         internal static void RegisterRefactorings<TCodeAction>(
             this CodeRefactoringContext context, ImmutableArray<TCodeAction> actions)
             where TCodeAction : CodeAction
+            => RegisterRefactorings(context, actions, applicableToSpan: null);
+
+        /// <summary>
+        /// Use this helper to register multiple refactorings (<paramref name="actions"/>) with an applicable span.
+        /// </summary>
+        internal static void RegisterRefactorings<TCodeAction>(
+            this CodeRefactoringContext context, ImmutableArray<TCodeAction> actions, TextSpan applicableToSpan)
+            where TCodeAction : CodeAction
+            => RegisterRefactorings(context, actions, new Nullable<TextSpan>(applicableToSpan));
+
+        private static void RegisterRefactorings<TCodeAction>(
+            CodeRefactoringContext context, ImmutableArray<TCodeAction> actions, TextSpan? applicableToSpan = null)
+            where TCodeAction : CodeAction
```

Comment:
why not just expose this and remove the other two?  they both seem to call into this and do nothing more beyond that.

Reply:
that was my original approach, but I noticed that the underlying call (registerrefactoring) did it this way - http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/CodeRefactorings/CodeRefactoringContext.cs,73
Which I am guessing is because we wanted either an explicit lack of value, or an explicit non-null value.  So I made these match.  Happy to change though if you would rather have just one

Reply:
> Happy to change though if you would rather have just one

I'd rather just have one.  It seems truly identical in this case :)